js/index.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-globals */
+class Books {
+    booklist = [];
+
+    addBooks() {
+      const name = document.getElementById('title').value;
+      const author = document.getElementById('author').value;
+      this.booklist.push({ title: name, author });
+      localStorage.setItem('books', JSON.stringify(this.booklist));
+      location.reload();
+    }
```

Comment:
- [ ] I can add books and they are being saved on local storage :+1:. But after I click the button the page shows an error. Kindly fix this
![image](https://user-images.githubusercontent.com/62705947/177794198-7217dbb0-7cbb-45ae-9696-7d9d89246724.png)

Reply:
https://www.loom.com/share/8c4bc90506ab448b8f860a5526c1e50f 

no such error pops up on my system,

Reply:
Hmm this is weird, please double check if you have pushed all the changes made on your local computer to Github. If so please ask for another code review because it could be that the problem was on my machine and the next reviewer can double-check it!